CONTRIBUTING.rst

```diff
@@ -328,6 +334,45 @@ detailing the test coverage::
   networkx/algorithms/approximation/clique.py         42      1     18      1    97%
   ...
 
+Adding tests
+------------
+
+If you're **new to testing**, see existing test files for examples of things to do.
+**Don't let the tests keep you from submitting your contribution!**
+If you're not sure how to do this or are having trouble, submit your pull request
+anyway.
+We will help you create the tests and sort out any kind of problem during code review.
+
+General guidelines for writing good tests:
+
+- doctests always assume ``import networkx as nx`` so don't add that
```

Comment:
This bullet might be a bit confusing for users who aren't as familiar with doctest since this is only relevant for docstring examples and not relevant for explicit tests. Maybe move to the end of the list, or have a separate section/callout on doctest? e.g.

```rst
.. note::
   Any code in the ``Examples`` section of docstrings is also run during testing
   via ``doctest``. Note that ``import networkx as nx`` is automatically run by
   doctest, so there is no need to include ``>>> import networkx as nx`` to
   the Examples
```

Reply:
I moved this to the reference guide and, for now, I think we can just assume if you are looking there you know what a doctest is.  I suspect not many people will look at the reference guide.  We can improve the text later.  Maybe a good way to do it would be to ask some (new?) contributors how much detail they need.  Or maybe they would prefer being pointed to an example, rather than having too much text.  Regardless, we can do that after 2.6.